README.adoc

```diff
@@ -71,27 +56,52 @@ different command line options for `docker-compose`, consult the
 https://docs.docker.com/compose/overview/[official docker-compose documentation].
 
 === Getting Started
-
 IMPORTANT: SMART COSMOS DevKit is designed for learning, testing, and evaluation
 of the SMART COSMOS platform. It is not intended for production use, and it is
-inherently insecure due to its reliance on a JKS key pair for which the
-https://github.com/SMARTRACTECHNOLOGY/smartcosmos-auth-server/blob/master/src/main/resources/smartcosmos.jks[secret key]
+inherently insecure due to its reliance on a JKS key pair for which the https://github.com/SMARTRACTECHNOLOGY/smartcosmos-auth-server/blob/master/src/main/resources/smartcosmos.jks[secret key]
 is publicly available and cannot be replaced, and its use of the primary Objects
 database as a store for user and account data. To establish a secure, production-ready
 SMART COSMOS environment, consult your SMARTRAC Technology Inc. sales representative.
 
 To jump right into operating the DevKit, follow our
-link:tutorials/getting-started.adoc[getting started tutorials].
+link:guides/getting-started.adoc[getting started tutorials].
 
-[[nextSteps]]
-== Next Steps
+== Open Source Framework
+SMART COSMOS includes a number of open source frameworks available under the
+Apache License, Version 2. You can clone and freely use these frameworks in your
+projects, submit pull requests, and access the latest documentation at GitHub:
+
+ * https://github.com/SMARTRACTECHNOLOGY/smartcosmos-framework
+
+ * https://github.com/SMARTRACTECHNOLOGY/smartcosmos-devkit
+(where this document was downloaded from)
+
+IMPORTANT: The core of the Objects server itself is presently proprietary and must
+be properly licensed when deployed in a production environment.
+
+In addition to the open source development framework, the entirety of the
+SMART COSMOS Objects REST Web Service APIs are fully documented at our https://api.smartcosmos.net[API Documentation web page] and the https://documenter.getpostman.com/view/437937/smart-cosmos-objects-devkit-api-tutorial/2JvFAy[SMART COSMOS Objects DevKit API Tutorial].
+
+[width="100%",options="header,footer"]
+|===
+|Service|API links|Configuration|Description
+|Authentication Server|https://api.smartcosmos.net/microservices/smartcosmos-auth-server/index.html[API]|https://github.com/SMARTRACTECHNOLOGY/smartcosmos-devkit/blob/master/config/smartcosmos-auth-server.yml[smartcosmos-auth-server.yml]| An authorization server to fit a flexible architecture for entirely different form of authentication
```

Comment:
Is it an _authentication_ server or an _authorization_ server? I think we should be consistent or avoid the distinction by calling it _auth_ server, but it's actually authentication server AFAIK.